sites/public/src/components/account/ApplicationsView.module.scss

```diff
@@ -0,0 +1,44 @@
+.applications-section-background {
+  background: var(--seeds-color-gray-300);
```

Comment:
Where possible it's preferable to use semantic tokens for theming, for example this could be `--seeds-bg-color-canvas` which is defaulted in our theme to `--seeds-color-gray-300`

Reply:
Oh nice, I haven't noticed the canvas one before!